platform/telemetry/src/TelemetryCapability.js

```diff
@@ -212,8 +212,10 @@ define(
             var telemetryAPI = this.openmct.telemetry;
 
             var metadata = telemetryAPI.getMetadata(domainObject);
-            var defaultDomain = (metadata.valuesForHints(['domain'])[0] || {}).key;
-            var defaultRange = (metadata.valuesForHints(['range'])[0] || {}).key;
+            var domainMetadata = metadata.valuesForHints(['x'])[0] || {};
```

Comment:
remove these ` || {};` because a) we expect all telemetry to have these values, and b) if this case is hit, this just defers the error to much later in the stack and it's harder to trace back to the root cause.